testkit/backend.py

```diff
@@ -34,7 +35,7 @@
             time.sleep(5)
             print("openning firefox")
             with open_proccess_in_driver_repo([
-                "firefox", "-headless", "http://localhost:8000"
+                "firefox", "-profile", "./testkit/firefox_profile", "-headless", "http://localhost:8000" # type: ignore
```

Comment:
```suggestion
                "firefox", "--profile", "./testkit/firefox_profile", "--headless", "http://localhost:8000" # type: ignore
```
(nit)
I mean this works as is, but if you do `firefox --help`, it'll show the full-name options prefixed with `--`